core/storage/statistics/gae_models.py

```diff
@@ -153,18 +184,18 @@ class MaybeLeaveExplorationEventLogEntryModel(base_models.BaseModel):
     Event schema documentation
     --------------------------
     V1:
-        event_type: 'leave' (there are no 'maybe leave' events in V0)
-        exploration_id: id of exploration currently being played
-        exploration_version: version of exploration
-        state_name: Name of current state
-        play_type: 'normal'
-        created_on date
-        event_schema_version: 1
-        session_id: ID of current student's session
-        params: current parameter values, in the form of a map of parameter
-            name to value
-        client_time_spent_in_secs: time spent in this state before the event
-            was triggered
+        event_type: 'leave' (there are no 'maybe leave' events in V0).
+        exploration_id: ID of exploration currently being played.
+        exploration_version: version of exploration.
+        state_name: Name of current state.
+        play_type: 'normal'.
+        created_on date: Date on which exploration was created.
```

Comment:
Actually, I think we should be removing this parameter everywhere. It's not a field of the model anymore if you take a look.

Reply:
I removed this everywhere.